src/GutenbergCleanup/Utils.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace WPML\PB\GutenbergCleanup;
+
+use WPML\FP\Relation;
+
+class Utils {
+
+	/**
+	 * @param string $content
+	 *
+	 * @return bool
+	 */
+	public static function hasGutenbergMetaData( $content ) {
```

Comment:
@strategio I think this is the same as the helper in WPML\LIB\WP.

```
WPML\LIB\WP\Gutenberg::hasBlock( $content );
```

Reply:
Thanks for reminding @brucepearson, I will reuse this and also move my `Utils::stripHtmlComments` to this library.

Reply:
Done (see https://git.onthegosystems.com/wpml-packages/wp/-/merge_requests/36)